gcloud/storage/_helpers.py

```diff
@@ -79,10 +79,12 @@ def _patch_property(self, name, value):
     def _set_properties(self, value):
         """Set the properties for the current object.
 
-        :type value: dict
+        :type value: dict or :class:`gcloud.storage.batch._FutureDict`
         :param value: The properties to be set.
         """
         self._properties = value
+        if hasattr(value, 'owner'):
+            value.owner = self
```

Comment:
This seems to be addressed. Chatted with @tseaver on Monday, April 20 and he said he finally grokked why this was necessary.